tensorflow/contrib/tensorrt/convert/convert_nodes_test.cc

```diff
@@ -2349,6 +2348,305 @@ TEST_F(OpConverterTest, ConvertSqueeze) {
   }
 }
 
+TEST_F(OpConverterTest, ConvertStridedSlice) {
+  {
+    // Input list is empty, should fail.
+    NodeDef node_def = MakeNodeDef("my_strided_slice", "StridedSlice", {});
+    RunValidationAndConversion(
+        node_def, error::INVALID_ARGUMENT,
+        "StridedSlice expects 4 inputs, at my_strided_slice");
+  }
+
+  // Get nodedef for StridedSlice layer.
+  auto get_strided_slice_nodedef = [](int begin_mask = 0, int ellipsis_mask = 0,
```

Comment:
optional: it is more intuitive to follow semantic order (end_mask before epllipsis_mask) as done in other places instead of using the alphabetical order.